crates/forge/tests/e2e/running.rs

```diff
@@ -854,14 +828,15 @@ fn printing_in_contracts() {
         Running 0 test(s) from src/
         Running 2 test(s) from tests/
         Hello world!
-        [PASS] tests::test_contract::test_increase_balance [..]
-        [PASS] tests::test_contract::test_cannot_increase_balance_with_zero_value [..]
+        [PASS] contract_printing_integrationtest::test_contract::test_increase_balance [..]
+        [PASS] contract_printing_integrationtest::test_contract::test_cannot_increase_balance_with_zero_value [..]
         Tests: 2 passed, 0 failed, 0 skipped, 0 ignored, 0 filtered out
         "#},
     );
 }
 
 #[test]
+#[ignore] //TODO unignore when there exists prvious version that supports new attributes
```

Comment:
don't understand this one

Reply:
This requires there to be previous version, but because we introduce now incompatibility with previous one, there must exists at least 2 versions with this new system to make this test work again

Reply:
Ah okay. Please leave a task though for this and mark it here by the TODO